label_studio/users/views.py

```diff
@@ -149,3 +158,28 @@ def user_account(request):
         'users/user_account.html',
         {'settings': settings, 'user': user, 'user_profile_form': form, 'token': token},
     )
+
+
+class UserSoftDeleteView(generics.RetrieveDestroyAPIView):
+    permission_required = ViewClassPermission(
+        DELETE=all_permissions.organizations_change,
+    )
+    queryset = User.with_deleted.all()
```

Comment:
This would allow soft-deleting a soft-deleted user a second time - any reason for that? I think the other manager might be a better fit here

Reply:
No I cant see a reason for that, bth not sure why I used that there.